modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessage.java

```diff
@@ -58,4 +67,14 @@ public void setMetadata(AndesMessageMetadata metadata) {
     public void addMessagePart(AndesMessagePart messagePart) {
         contentChunkList.add(messagePart);
     }
+
+    /**
+     * Check whether message should be delivered to given subscriber
+     * @param subscription message receiver subscription information
+     * @return
+     */
+    public boolean isDelivarable(AndesSubscription subscription)
+    {
+        return true;
```

Comment:
Should it always return 'true' ? shouldn't it be the case where we need to process the param passed in to check the deliverable status